app/services/AlfrescoStore.java

```diff
@@ -111,6 +117,18 @@ public AlfrescoStore(Config configuration, WSClient wsClient) {
                 });
     }
 
+    @Override
+    public CompletionStage<Boolean> isHealthy() {
+        return wsClient.url(alfrescoUrl + "noms-spg/")
```

Comment:
Use makeRequest() below for DRY code

Reply:
Did look at that but makeRequest uses the `onBehalfOfUser` which we don't have in the context of a health endpoint been called